lib/main.coffee

```diff
@@ -99,6 +99,14 @@ module.exports =
       default: 'Fuzzy'
       enum: ['Fuzzy', 'Symbol']
       order: 15
+    suppressActivationForEditorClasses:
+      title: 'Suppress Activation For Editor Classes'
+      description: 'Don\'t auto-activate when any of these classes are present in the editor.'
+      type: 'array'
+      default: ['vim-mode.insert-mode']
```

Comment:
https://github.com/atom-community/autocomplete-plus/pull/297/files#diff-e0081a61a70bd2d693ae6009aea42e5eR275

You're right, I've represented the inverse of what is desired, right?